Repairs/app/src/main/java/com/rooio/repairs/Jobs.kt

```diff
@@ -1,15 +1,52 @@
 package com.rooio.repairs
 
+import android.content.Intent
 import android.os.Bundle
+import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ListView
+import androidx.constraintlayout.widget.ConstraintLayout
 import androidx.transition.TransitionManager
+import org.json.JSONArray
+import org.json.JSONObject
+import java.util.ArrayList
+import androidx.arch.core.util.Function
+
 
 class Jobs : NavigationBar() {
+    private var id1: ListView? = null
+    private var id2: ListView? = null
+    private var id3: ListView? = null
+    private var completedButton: Button? = null
+
+
+    val statuses = arrayListOf<String>()
+
+    private var pendingConstraint: ConstraintLayout? = null
+    private var scheduledConstraint: ConstraintLayout? = null
+    private var inProgressConstraint: ConstraintLayout? = null
+
+    companion object{
+        @JvmStatic private var pendingJobs = ArrayList<JSONObject>()
+        @JvmStatic private var scheduledJobs = ArrayList<JSONObject>()
+        @JvmStatic private var inProgressJobs = ArrayList<JSONObject>()
+        @JvmStatic private var archivedJobs = ArrayList<JSONObject>()
+    }
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_jobs)
 
         //sets the navigation bar onto the page
+        id1 = findViewById<View>(R.id.id1) as ListView
```

Comment:
We're putting the navigation bar setup into its own function now. Check mine or Luke's pages to see how it should look